peewee_async_compat.py

```diff
@@ -0,0 +1,442 @@
+"""
+Compatibility layer for `peewee_async` to navigate smooth migration towards v1.0.
+
+In the initial implementation the `Manager` class was introduced to avoid models
+subclassing (which is not always possible or just undesirable). The newer interface
+relies more to models subclassing, please check the `peewee_async.AioModel`.
+
+Licensed under The MIT License (MIT)
+
+Copyright (c) 2024, Alexey Kinëv <rudy@05bit.com>
+
+"""
+from functools import partial
+
+import warnings
+import peewee
+
+IntegrityErrors = (peewee.IntegrityError,)
+
+try:
+    import aiopg
+    import psycopg2
+    IntegrityErrors += (psycopg2.IntegrityError,)
+except ImportError:
+    aiopg = None
+    psycopg2 = None
+
+__all__ = [
+    'Manager',
+    'count',
+    'prefetch',
+    'execute',
+    'scalar',
+]
+
+
+def _patch_query_with_compat_methods(query, async_query_cls):
+    """
+    Patches original peewee's query with methods from AioQueryMixin, etc.
+
+    This is the central (hacky) place where we glue the new classes with older style
+    `Manager` interface that operates on original peewee's models and query classes.
+
+    Methods added to peewee's original query:
+
+    - aio_execute
+    - fetch_results
+    - make_async_query_wrapper
+    - aio_get (for SELECT)
+    - aio_scalar (for SELECT)
+    """
+    from peewee_async import AioModelSelect
```

Comment:
Will you please add a condition:

```python
from peewee_async import AioModelSelect, AioModel
if isinstance(query.model, AioModel):
        return
```
It let us not to patch if our model is inherited from Ai Model and will be more reliable than use hacky patches

Reply:
Found you have already did it here  #233 I will resolved the commend when #233 is merged

Reply:
@kalombos Merged it here, let's do a final review and merge :)